go/vt/sqlparser/tracked_buffer_test.go

```diff
@@ -188,6 +188,10 @@ func TestCanonicalOutput(t *testing.T) {
 			"create table t (p point srid 0, g geometry not null srid 4326)",
 			"CREATE TABLE `t` (\n\t`p` point SRID 0,\n\t`g` geometry NOT NULL SRID 4326\n)",
 		},
+		{
+			input:     "select regexp_replace('abc def ghi', '[a-z]+', 'X', 1, 3, 'c'), REGEXP_LIKE('dog cat dog', 'dog'), REGEXP_SUBSTR('abc def ghi', '[a-z]+', 1), REGEXP_INSTR('aa aaa aaaa aaaa aaaa aaaa', 'a{4}',1), 'Michael!' RLIKE '.*' from dual",
+			canonical: "SELECT REGEXP_REPLACE('abc def ghi', '[a-z]+', 'X', 1, 3, 'c'), REGEXP_LIKE('dog cat dog', 'dog'), REGEXP_SUBSTR('abc def ghi', '[a-z]+', 1), REGEXP_INSTR('aa aaa aaaa aaaa aaaa aaaa', 'a{4}', 1), 'Michael!' REGEXP '.*' FROM `dual`",
```

Comment:
All other cases here don't use the prefix but depend on ordering. I'm more of a fan of including the names, but even more a fan of staying consistent with the rest :smile:. So I think we should stay consistent here.

Reply:
Can you also add a case here for the `NOT` (also in reference to my earlier comment of using `astPrintf` for that.

Reply:
> All other cases here don't use the prefix but depend on ordering. I'm more of a fan of including the names, but even more a fan of staying consistent with the rest smile. So I think we should stay consistent here.

I did not understand what needs to be done here. :sweat_smile:

Reply:
@K-Kumar-01 You can drop the `input:` and `canonical:` prefixes here, so it looks like entries above.